docs/src/active-layer-transport.md

```diff
@@ -366,12 +366,13 @@ transport_test_input(;
 	initial_topography = (x, y) -> 0.0u"m",
 	initial_sediment = (x, y) -> 0.0u"m",
 	disintegration_rate = 50.0u"m/Myr",
-	subsidence_rate = 50.0u"m/Myr",
+	subsidence_rate = 0.0u"m/Myr",
```

Comment:
Why 50 ---> 0 ?

Reply:
The default value is 0, and this value is not yet used in any of the tests.